docs/moduleAna5Utils.rst

```diff
@@ -221,3 +223,15 @@ It is also necessary to extend the profile in the runout area. This is done by f
 direction of the path given by the few last points in the path in (x,y) (all points at a distance
 ``nCellsMinExtend`` x cellSize < distance < ``nCellsMaxExtend`` x cellSize)) and extending in
 this direction for a given percentage (``factBottomExt``) of the total length of the path :math:`s`.
+
+
+Split point generation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A parabolic curve is fitted to the avalanche path profile extracted from the DFA simulation (non-extended profile).
+The first and last points of the parabolic profile match the avalanche path profile. To find the
+best fitting parabolic profile, a constrain is needed. Two options are available:
+the default one (``fitOption``= 0) minimises the distance between the parabolic profile and the avalanche path
+profile. The second option (``fitOption``= 1) matches the end slope of the profile to the parabola.
```

Comment:
The second option tries to match the parabola to the end slope of the profile - is it like this? I mean it uses the profile as input to then create the parabola right? So the profile is not changed